### DISCUSSION TITLE
Avatar group component for logos

### DISCUSSION THREAD

**Discussion by char-mainewang** [twilio-labs/paste#3305] [General] [answered]
Hey team, in situations where we want to display a stack of logos (i.e. integration logos in Segment), should we be 
1. using the icon variation, use the icons in the Segment design library and use the default color background 
2. or use the image variation by creating image components of each logo
<img width="290" alt="image" src="https://github.com/twilio-labs/paste/assets/87334135/559db530-6227-4fb7-a356-043151bb8802">

**Comment by serifluous** [comment, marked as answer]
Hey @char-mainewang !

If you have non-Twilio/Segment logos, you should use the image variant. The icon variant will only accept Paste icons.

I'm going to try and see if we can modify the structure of the Avatar component in Figma so you can use your logo components directly from the Segment Design Library, without having to upload them each time into it.

**Reply by serifluous** [reply]
Ok! So it's not a perfect fix, but I merged in an update to the Figma Avatar component so it works _enough_ with the logos from the Segment Design Library for now.

You can see an example here: https://www.figma.com/file/SpeLpBoqN2vZv8KWcSn0J1/Paste-Feedback?type=design&node-id=2712-609906&mode=design&t=ZhTNJ2DKcYh4ANDg-0

For now, to get it to work well with all sizes of Avatar, you'll have to add padding to the image on the logo component itself (like the Adobe Analytics logo y'all already have). Or wait until we figure out a solution in the Avatar component, which I've backlogged: https://issues.corp.twilio.com/browse/DSYS-3970
